src/Algebra/Properties/CommutativeMonoid.agda

```diff
@@ -0,0 +1,181 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some derivable properties
+------------------------------------------------------------------------
+
+open import Algebra
+
+module Algebra.Properties.CommutativeMonoid {g₁ g₂} (M : CommutativeMonoid g₁ g₂) where
+
+open CommutativeMonoid M renaming (ε to 0#; _∙_ to _+_; ∙-cong to +-cong; identity to +-identity; assoc to +-assoc; comm to +-comm)
```

Comment:
This seems quite key, and at the moment it's difficult to spot. Could you open this at the end of the imports, and lay each renaming instance out on a separate line?